src/xdocs/config_imports.xml

```diff
@@ -706,7 +706,9 @@ import android.*;
           <tr>
             <td>file</td>
             <td>
-              name of the file containing the import control configuration.
+              The location of the file containing the import control configuration.
+              It can be a regular file, URL or resource path. It will try loading
+              the path as a file first, then as a resource.
```

Comment:
url first, then as file, then as resource.
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java#L362

please be exact. It is critical nuance for user.